docs/user/openstack/README.md

```diff
@@ -68,13 +74,15 @@ openstack role add --user <user> --project <project> swiftoperator
 openstack object store account set --property Temp-URL-Key=superkey
 ```
 
-**NOTE:** Swift is required as the user-data provided by OpenStack is not big enough to store the ignition config files, so they are served by Swift instead.
+**NOTE:** Swift is required as the user-data provided by OpenStack Metadata service is not big enough to store the ignition config files, so they are served by Swift instead.
 
-* You may need to increase the security group related quotas from their default   values. For example (as an OpenStack administrator):
+For a successful installation it is required:
 
-```sh
-openstack quota set --secgroups 8 --secgroup-rules 100 <project>`
-```
+* Containers: 2 (one for the image registry and another one for ignition files)
+* Objects: 1
+* Available space in Swift: 10 MB
```

Comment:
While 10 MB is sufficient quota for installation, it's not enough for any meaningful work with the cluster. You should probably add a note somewhere in this doc.

Reply:
Yeah, it is not clear that here we're talking about installation only, where we need Swift just to store the ignition file. Typically its size is < 300 kb, so even 300 kb is sufficient.
I think I should update the document to add these details.